lib/compat/wordpress-6.1/class-wp-theme-json-6-1.php

```diff
@@ -47,6 +47,15 @@ class WP_Theme_JSON_6_1 extends WP_Theme_JSON_6_0 {
 		'caption' => 'wp-element-caption',
 	);
 
+	// List of block support features that can have their related styles
+	// generated under their own feature level selector rather than the block's.
+	const SUPPORT_FEATURES = array(
```

Comment:
Probably a nitpick:

The comment is pretty clear, but I'm thinking about whether `SUPPORT_FEATURE` describes the role of the whitelist to folks digging about in the code.

Would `BLOCK_SUPPORT_FEATURES` or even `BLOCK_SUPPORT_FEATURE_LEVEL_SELECTORS` add anything?